test/example.js

```diff
@@ -162,6 +162,12 @@ const functionWithFunction = (fn, number) => fn(Math.random() * number);
 
 const arrayExportExample = (num1, num2) => {};
 
+const __SOME_DEFAULT__VALUE__ = 42;
```

Comment:
```suggestion
const __SOME_DEFAULT__VALUE_ = 42;
```
to make sure we cover single trailing underscore?